lib/vty.c

```diff
@@ -281,7 +281,8 @@ int vty_out(struct vty *vty, const char *format, ...)
 	return len;
 }
 
-int vty_json(struct vty *vty, struct json_object *json)
+static int vty_json_helper(struct vty *vty, struct json_object *json,
+			   uint32_t options)
```

Comment:
I don't see that `options` is really used in the function.
`json_object_to_json_string_ext` is still called with `JSON_C_TO_STRING_PRETTY | JSON_C_TO_STRING_NOSLASHESCAPE`.

Reply:
Yeah, we are waiting for https://github.com/FRRouting/frr/pull/12734 and I'm going to append this.